app/_src/gateway/kong-enterprise/secrets-management/secrets-rotation.md

```diff
@@ -30,7 +30,7 @@ Kong PDK API that can be used to rotate secrets on failure: [kong.vault.try](/ga
 
 ## Periodically rotating secrets using TTLs
 
-Kong employs a background job that runs periodically on **every minute** (thus you cannot currently
+Kong employs a background job that runs periodically **once every minute** (thus you cannot currently
 automatically rotate more often than that). Its job is to rotate secrets that are about to expire.
```

Comment:
Another attempt:

> Kong operates a background job that executes every minute, serving the function of rotating secrets that are nearing expiration. As such, it is not possible to automatically rotate secrets more frequently than this interval. This characteristic may create challenges, especially for TTLs that are set for less than one minute. The once-per-minute refresh rate means that a secret with a TTL of 30 seconds might take up to 60 seconds to refresh in the least favorable scenario. This delay arises from the alignment of the secret's TTL with the background job's minute-long execution cycle, potentially doubling the expected refresh time. Additionally, if the sum of the TTL and the resurrect_ttl is less than 60 seconds, it can lead to issues since the secret may not be refreshed or resurrected within the designated timeframe, impacting the continuity of operations that depend on the secret.